src/main/java/org/folio/marccat/dao/persistence/REF.java

```diff
@@ -138,38 +142,49 @@ public void init(Descriptor source, Descriptor target, int referenceType, int ca
    * Sets the default.
    */
   public void setDefault() {
+    this.setPrintConstant('n');
     this.setAuthorityStructure('a');
     this.setEarlierRules('x');
     this.setFormerHeading('x');
-    if (getKey() != null
-      && ReferenceType.isEquivalence(getKey().getType())) {
+    if (getKey() != null && ReferenceType.isEquivalence(getKey().getType())) {
       this.setNoteGeneration('x');
     } else {
-      this.setNoteGeneration('@');
+      this.setNoteGeneration('n');
     }
```

Comment:
Certainly not required, but consider using a ternary expression here, e.g.
```
    this.setNoteGeneration(
        getKey() != null && ReferenceType.isEquivalence(getKey().getType()) ? 'x' : 'n');
```

Reply:
I have changed. Thanks a lot.